assets/sass/style.scss

```diff
@@ -1,7 +1,99 @@
 @import 'reset';
+html {
+  height: 100%;
+}
 body {
   margin: 0;
+  height: 100%;
+  width: 100%;
 }
 .main {
   width: 100%;
+  height: 100%;
+  overflow: hidden;
+}
+header {
+  height: 75px;
+  position: fixed;
+  top: 0;
+  background-image: #2B3C43 url("../images/header-right.png") no-repeat right;
+  background-size: 268px;
+  width: 100%;
+  span {
+    color: #D1D3D5;
+    position: relative;
+    top: 30%;
+    left: 45%;
+  }
+}
+footer {
```

Comment:
Use positioning only if it's necessary. Use margin and other properties instead.